proto/vtctldata.proto

```diff
@@ -2213,3 +2213,37 @@ message WorkflowMirrorTrafficResponse {
   string start_state = 2;
   string current_state = 3;
 }
+
+message LookupVindexCreateParams {
+  // Keyspace to create the Lookup Vindex(es) in.
+  string keyspace = 1;
+  // Vindex(es) configuration.
+  repeated VindexParams vindexes = 2;
+
+  // VReplication specific options.
+  repeated string cells = 3;
+  repeated topodata.TabletType tablet_types = 4;
+  optional bool continue_after_copy_with_owner = 5;
+  optional bool tablet_types_in_preference_order = 6;
+}
+
+message VindexParams {
+  // Name of the Lookup Vindex to create.
+  string name = 1;
+  // Type of Lookup Vindex to create.
+  string lookup_vindex_type = 2;
+  // Table holding the data which we should use to backfill the Lookup Vindex.
+  // This must exist in the same keyspace as the Lookup Vindex.
+  string table_owner = 3;
+  // Columns to read from the owner table.
+  repeated string table_owner_columns = 4;
+  // Name of the lookup table. If not specified, then it will be created using
+  // the same name as the Lookup Vindex.
+  string table_name = 5;
+  // Do not add corresponding records in the lookup table if any of the owner
+  // table's 'from' fields are NULL.
+  bool ignore_nulls = 6;
+  // The primary vindex name/type to use for the lookup table,
+  // if the table-keyspace is sharded.
+  string table_vindex_type = 7;
+}
```

Comment:
These shouldn't be proto messages as they are not sent over the wire. We should be making any needed modifications to `LookupVindexCreateRequest` and the other things can be normal in-memory structs with JSON tags.

Reply:
Why can't we simply add JSON tags to the existing struct?
```
	createOptions = struct {
		Keyspace                     string
		Type                         string
		TableOwner                   string
		TableOwnerColumns            []string
		TableName                    string
		TableVindexType              string
		Cells                        []string
		TabletTypes                  []topodatapb.TabletType
		TabletTypesInPreferenceOrder bool
		IgnoreNulls                  bool
		ContinueAfterCopyWithOwner   bool
		ParamsFile                   string
	}{}
```

Reply:
This is what we should do IMO. Then we're filling in the same struct via a combination of flags and JSON fields in a file. And we could allow both with command-line flags taking precedence.

Reply:
Removed the proto messages. I think it will be a little complex to configure json file if we use the `createOptions` for multiple lookup vindexes, even if we modify it by using slices (e.g. using `Type` as `[]string` to support configuring multiple vindexes.) So, created this struct for unmarshalling:
```go
// vindexParams is used to unmarshal content from params-file.
type vindexParams struct {
	LookupVindexType  string   `json:"lookup_vindex_type"`
	TableOwner        string   `json:"table_owner"`
	TableOwnerColumns []string `json:"table_owner_columns"`
	TableName         string   `json:"table_name"`
	TableVindexType   string   `json:"table_vindex_type"`
	IgnoreNulls       bool     `json:"ignore_nulls"`
}
```

Please let me know your views on this.